src/ride/ride_data.c

```diff
@@ -1729,3 +1729,508 @@ const rating_tuple RideRatings[91] = {
 	{	50,		30,	30	},	// RIDE_TYPE_59
 	{	50,		30,	10	},	// RIDE_TYPE_LIM_LAUNCHED_ROLLER_COASTER
 };
+
+// rct2: 0x0097CF40
+const rct_ride_properties gRideProperties[91] = {
```

Comment:
Why are some constants prefixed with `g`? I'd expect it to be for global *variables* only.